### ISSUE TITLE
Change markdown-code-fontification buffer names to use "earmuffs"

### ISSUE THREAD

**Issue by shipmints** [jrblevin/markdown-mode#845]
It would be nicer for markdown temporary buffers to have earmuffs as many people use conventional buffer filters that are set to filter buffers with names that conform to ```" *xxx*"```.

```elisp
(defun markdown-fontify-code-block-natively (lang start end)
...
        (with-current-buffer
            (get-buffer-create
             (concat " markdown-code-fontification:" (symbol-name lang-mode)))
;; better as 
             (concat " *markdown-code-fontification:" (symbol-name lang-mode) "*")))
```

**Comment by syohex** [comment]
fixed #851

**Comment by shipmints** [comment]
Great. Thank you. Nice to know this package is being maintained.
